public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,316 @@
+  Angular 2 is running in the development mode. Call enableProdMode() to enable the production mode.
+
+:marked
+  To enable production mode, add this code to the start of your `main.ts`:
+
+code-example().
+  import { enableProdMode } from '@angular/core';
+  enableProdMode();
+
+:marked
+  ### Lazy loading
+
+  The Angular Router allows you to configure NgModules as being 
+  [Lazy Loaded](https://angular.io/docs/ts/latest/guide/router.html#!#asynchronous-routing),
+  which means that particular NgModule (and all it's code) is not loaded on the very first load.
+  This allows your app to have a smaller initial load time by not loading modules until they
+  are needed.
+
+  A common mistake to make while lazy loading a module is to *also* have that module imported via
+  a ES6 import. 
+  If you do that, the module will never really be lazy loaded since the import statement will cause
+  it to be imported directly instead of being imported via the Router. 
+
+  It's important to note that your bundling configuration must take lazy loading into consideration.
+  Since there is no ES6 import for that module, normal bundlers don't know that they should also
+  separately bundle to module listed in the router config.
+  You have to manually create additional bundles for each lazy loaded route.
+
+  If you are using Webpack, the 
+  [Angular Ahead-of-Time Webpack Plugin](https://github.com/angular/angular-cli/tree/master/packages/%40ngtools/webpack)
+  will automatically recognize lazy routes.
+
+
+.l-main-section
+:marked
+  ## Server configuration
+
+  Angular apps are 
+  [Single Page Applications](https://en.wikipedia.org/wiki/Single-page_application) (SPAs),
+  which makes them the perfect candidates to be served by simple static HTML server.
+  No preprocessors required!
+
+  There is only one configuration item required of a server hosting an Angular app:
+  it needs to be able to fallback to `index.html` when being asked for a file the server
+  does not have.
+
+  ### Why fallback to `index.html`?
+
+  When using the Angular Router, it is the one in charge of the routing instead of your server.
+  Angular routing defaults to 
+  [HTML 5 pushState](https://angular.io/docs/ts/latest/guide/router.html#!#browser-url-styles),
+  where it manipulates the browser address and history directly.
+  So, while the server has the responsability of serving the files, it is always the Angular app
+  that will determine which routes are valid and construct the page accordingly.
+  
+  Imagine you have the `admin/` route for your `www.my-awesome-app.com` Angular app. 
+  When you visit `www.my-awesome-app.com` most servers will respond with
+  the `index.html` located at `www.my-awesome-app.com/index.html`. 
+  Then you click the link for `admin/` and the your address bar will change 
+  to `www.my-awesome-app.com/admin/` and you're taken to the admin page.
+
+  But if you don't have fallback to `index.html` configured and refresh the page, you will
```

Comment:
To make this more clear, consider rearranging this sentence. "If you then refresh the page, you will get a 404 - Not Found page unless you have the fallback to index.html configured."